ledger-core/crypto/legacyadapter/digest_256.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package legacyadapter
+
+import (
+	"io"
+
+	"github.com/insolar/assured-ledger/ledger-core/cryptography"
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/cryptkit"
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/longbits"
+)
+
+const SHA3Digest256 = cryptkit.DigestMethod("sha3-256")
+
+type Sha3Digester256 struct {
+	scheme cryptography.PlatformCryptographyScheme
+}
+
+func NewSha3Digester256(scheme cryptography.PlatformCryptographyScheme) *Sha3Digester256 {
+	return &Sha3Digester256{
+		scheme: scheme,
+	}
+}
+
+func (pd *Sha3Digester256) DigestData(reader io.Reader) cryptkit.Digest {
+	hasher := pd.scheme.IntegrityHasher()
+
+	_, err := io.Copy(hasher, reader)
+	if err != nil {
+		panic(err)
+	}
+
+	bytes := hasher.Sum(nil)
+	bits := longbits.NewBits256FromBytes(bytes[:32])
```

Comment:
I would move '32' to constant and refuse it here and below

Reply:
1. this is legacy adapter to be removed
2. a string constant for 256 bits = 32 bytes ... does it make any sense?